Justfile

```diff
@@ -50,7 +50,12 @@ build-generate:
 build +tools: build-protos build-zips build-frontend
   #!/bin/bash
   shopt -s extglob
-  for tool in $@; do mk "{{RELEASE}}/$tool" : !(build|integration) -- go build -o "{{RELEASE}}/$tool" -tags release -ldflags "-X github.com/TBD54566975/ftl.Version={{VERSION}} -X github.com/TBD54566975/ftl.Timestamp={{TIMESTAMP}}" "./cmd/$tool"; done
+
+  if [ "${DEBUG:-}" = "true" ]; then
```

Comment:
`DEBUG=true just build ftl` will build with debug info. it's needed for the debugger but apparently creates suboptimal performance for the binary, so gating it to insulate the actual release binary